### ISSUE TITLE
Generation of CL bridge files is broken

### ISSUE THREAD

**Issue by gouttegd** [obophenotype/uberon#2143]
For each foreign ontology _X_ with which we have mappings, the pipeline is supposed to generate two bridge files in `src/ontology/bridges`: one for mappings with Uberon (`src/ontology/bridges/uberon-bridge-to-X`) and one for mappings with CL (`src/ontology/bridges/cl-bridge-to-X`).

What is currently happening is that only the `uberon-bridge-to-X` file is generated, containing both the Uberon mappings _and_ the CL mappings. The `cl-bridge-to-X` is not generated.

Not sure I fully understand why that is the case, but here are some paths to investigate:

The bridge files are generated by the `make-bridge-ontologies-from-xrefs.pl` script, which takes as input an OBO file and produces bridge files according to instructions of the form `treat-xrefs-as...` that it finds in the source file (for example, a line`treat-xrefs-as-reverse-genus-differentia: FBbt part_of NCBITaxon:7227` instructs the script to generate a bridge file from FBbt cross-references). Such instructions are present in `uberon-edit.obo`, but _not_ in CL, which must be why the CL bridges are not generated.

As for why the Uberon bridges contain axioms for both Uberon and CL (instead of Uberon only), this is seemingly because the file given as input to the `make-bridge-ontologies-from-xrefs.pl` script (`$(TMPDIR)/seed.obo`) is ultimately the result of merging `uberon-edit.obo` with a version of CL (`$(TMPDIR)/cl-core.obo`). I am not sure I understand why this merging is necessary.

**Comment by gouttegd** [comment]
Update: the reason why the CL bridges are not generated is in this rule:

```make
$(TMPDIR)/cl-with-xrefs.obo: $(TMPDIR)/cl-core.obo $(SCRIPTSDIR)/expand-idspaces.pl
	if [ $(BRI) = true ]; then egrep '^(idspace|treat-)' $(SRC) > $@ && cat $< >> $@.tmp && $(SCRIPTSDIR)/expand-idspaces.pl $@.tmp > $@; fi
```

That rule is intended to create a version of CL containing all the `treat-xrefs-as...` stanzas copied over from `uberon-edit.obo`. However, this is not what it does, because the `treat-xrefs-as...` lines are sent to `$@` instead of `$@.tmp`, and `$@` is then completely overwritten by the result of the `expand-idspaces.pl` script.

This is an easy one to fix and I’ll have a PR for that tomorrow (once I have checked my fix does not break anything else…).

That still leaves the problem that the Uberon bridges contain axioms for both Uberon and CL.

**Comment by gouttegd** [comment]
Fixed by the combination of #2132 and #2147.
